levels/inside_perimeter/maps/courtyard.json

```diff
@@ -0,0 +1,309 @@
+{ "compressionlevel":-1,
```

Comment:
I noticed that the name for this map is `courtyard`, but the png is named `courtyardv2`. Don't know that it would break anything, but thought I'd mention it